src/todolist.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] Please remove these eslint disables and fix any issues that appear. The ESlint warnings usually are a sign of badly organized code so it's best to try to fix these issues.